llvm/include/llvm/ADT/EquivalenceClasses.h

```diff
@@ -78,11 +78,13 @@ template <class ElemTy> class EquivalenceClasses {
     // ECValue ctor - Start out with EndOfList pointing to this node, Next is
     // Null, isLeader = true.
     ECValue(const ElemTy &Elt)
-      : Leader(this), Next((ECValue*)(intptr_t)1), Data(Elt) {}
+        : Leader(this), Next((ECValue *)(intptr_t)1), Data(Elt) {}
 
     const ECValue *getLeader() const {
-      if (isLeader()) return this;
-      if (Leader->isLeader()) return Leader;
+      if (isLeader())
+        return this;
+      if (Leader->isLeader())
+        return Leader;
```

Comment:
Please avoid unrelated changes.

Reply:
+1, you can land a separate clean-up PR (NFC) that fixes formatting

Reply:
You should be able use clang-format-diff or apply the diff suggested by the format checker